src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -690,7 +712,7 @@ else if (useFilteredViolations) {
         }
     }
 
-    private static List<TestInputViolation> getExpectedViolationsForSpecificLineAbove(
+    private static List<TestInputViolation> getExpectedViolationsForSpecificLine(
```

Comment:
Renamed this method as it is not specific to only `Lines Above` messages anymore